pkg/framework/plugins/removepodsviolatinginterpodantiaffinity/pod_antiaffinity.go

```diff
@@ -58,6 +58,7 @@ func New(args runtime.Object, handle framework.Handle) (framework.Plugin, error)
 	}
 
 	podFilter, err := podutil.NewOptions().
+		WithFilter(handle.Evictor().PreEvictionFilter).
```

Comment:
`Deschedule` method is using `d.handle.Evictor().Filter`. `handle.Evictor().PreEvictionFilter` needs to be combine there:
```
for i := 0; i < totalPods; i++ {
	if checkPodsWithAntiAffinityExist(pods[i], pods) && d.handle.Evictor().Filter(pods[i]) {
		if d.handle.Evictor().Evict(ctx, pods[i], evictions.EvictOptions{}) {
```